### PR TITLE
Pass in stock_location_quantities to order contents

### PR DESCRIPTION
**Pull request by exsuzme**
This is required to complete functionality of: https://github.com/bonobos/spree/pull/175

### CODE REVIEW

**Comment by athal7** [comment]
can we do a *args instead since we don't use these args and so that we don't run into this in the future

**Comment by jordan-brough** [comment]
ditto on what andrew said

**Comment by athal7** [comment]
:+1:

File: app/models/spree/order_contents_decorator.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Spree::OrderContents.class_eval do
-  def add_with_avatax(variant, quantity = 1, currency = nil, shipment = nil)
-    add_without_avatax(variant, quantity, currency, shipment).tap do
+  def add_with_avatax(*args)
+    add_without_avatax(args).tap do
```
Comment (jordan-brough) [inline app/models/spree/order_contents_decorator.rb]: need to use the `*` here also. `add_without_avatax(*args).tap do`

**Pull request merged**